class01/homework/KimJinho/hw4/README.md

```diff
@@ -0,0 +1,14 @@
+# Class01 HW4 Style Transfer
+
+## Create virtual environment (.hw4)
+
+## Environment
+Ubuntu 22.04
+Python 3.10.12
+Openvino 2023.1.0
+
+## Setting
+```sh
+pip install -r requirments.txt
+python3 hw4.py
+```
```

Comment:
This is a minor correction. It will be better to separate "## setting part" and "## how to run" part or just refer to it as "## how to run".
"## how to run" seems to be a more user-friendly or commonly used term.